init.php

```diff
@@ -1,13 +0,0 @@
-<?php
-
-/**
- * @deprecated since 3.0
- */
-define( 'WIKIBASE_DATAMODEL_JAVASCRIPT_VERSION', '4.0.0' );
```

Comment:
We usually mention the removal of constants in the release notes.

Reply:
hm, can say, although it says already there is only JS now there (thus no PHP constants).